zerver/lib/url_preview/preview.py

```diff
@@ -49,7 +48,7 @@ def guess_mimetype_from_content(response: requests.Response) -> str:
 
 def valid_content_type(url: str) -> bool:
     try:
-        response = requests.get(url, stream=True, headers=HEADERS, timeout=TIMEOUT)
+        response = requests.get(mark_sanitized(url), stream=True, headers=HEADERS, timeout=TIMEOUT)
```

Comment:
What is this for?

Reply:
I think I copy paste the same `response = requests.get(mark_sanitized(url), stream=True, headers=HEADERS, timeout=TIMEOUT)` at line number 91 and 51.
I'll correct this.